examples/ibis/models/ibis_full_model.py

```diff
@@ -0,0 +1,19 @@
+import ibis
+
+from sqlmesh.core.macros import MacroEvaluator
+from sqlmesh.core.model import model
+
+
+@model("ibis.ibis_full_model", is_sql=True, kind="FULL", audits=["assert_positive_order_ids"])
+def entrypoint(evaluator: MacroEvaluator) -> str:
+    # connect to database
+    con = ibis.duckdb.connect(database="data/local.duckdb")
+
+    # retrieve table
+    incremental_model = con.table("incremental_model", schema="ibis")
```

Comment:
you need to get the table from the evaluator so that dependencies are properly tracked.

table_name = context.table("ibis.incremental_model")